modules/workflow-service-impl/src/main/java/org/opencastproject/workflow/impl/WorkflowOperationWorker.java

```diff
@@ -169,7 +169,7 @@ public WorkflowInstance execute() {
       workflow = service.handleOperationResult(workflow, result);
       return workflow;
     } catch (JobCanceledException e) {
-      logger.info(e.getMessage());
+      logger.info("Job cancelled: {}", e.getMessage());
       return workflow;
```

Comment:
It would be useful to add the workflow Id

Reply:
Added